packages/datadog-plugin-jest/test/circus.spec.js

```diff
@@ -312,6 +279,98 @@ describe('Plugin', function () {
           )
         })
       }
+
+      describe('agentless', () => {
```

Comment:
What is the distinction between agentless and not agentless? I thought the feature needed to always be agentless in the first place?

Reply:
this is just for convenience: in this describe we have tests that require agentless mode. This is so that we can easily identify them and modify the inputs to `agent.load`